Makefile

```diff
@@ -1,39 +1,82 @@
-.PHONY: build
+# Set the default target to build all
+.DEFAULT_GOAL := all
+
+# Declare all targets as phony targets
```

Comment:
Is default help better?

Reply:
Yes, I think the default help information is better